websites/N/Navidrome/presence.ts

```diff
@@ -0,0 +1,42 @@
+const presence = new Presence({
+	clientId: "1126923786600583339",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/mhD2eIr.png",
+	};
+
+	// Determine player state
+	let playerState;
+	// Check if audio-title has any content
+	if (document.querySelectorAll(".audio-title")[0].innerHTML !== "") {
```

Comment:
Prefer `.textContent` over `.innerHTML` where possible.
```suggestion
	if (document.querySelector(".audio-title").textContent) {
```